app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,51 @@
+require 'date'
+# Public
+# Controls the attendance of employees
+# It does the following tasks:
+#   - Mark attendance of a employee/ Employee
+#   - Let admin see who is present today
+class AttendancesController < ApplicationController
+  def index
+    @company = Company.find(session[:current_company_id])
+    @attendances_list = @company.attendances.where(status: 1).order(login_time: :desc)
+  end
+
+  def create
+    @current_employee = Employee.find(session[:current_employee_id])
```

Comment:
This code can be better, when cancancan apply in this project.. (future comment :P)

Reply:
yes, this is here just because it isn't merged with other